pom.xml

```diff
@@ -44,7 +44,7 @@
         <hamcrest.version>1.3</hamcrest.version>
         <mockito.version>2.13.0</mockito.version>
         <jest.version>6.3.1</jest.version>
-        <test.containers.version>1.11.4</test.containers.version>
+        <test.containers.version>1.15.0-rc2</test.containers.version>
```

Comment:
is there a specific feature we are looking for in the 1.15 and we are getting the release candidate? Or the previous one is good enough?

Reply:
the latest mac OS update broke test containers - `1.15.0-rc2` is the only version that contains this fix and was released by testcontainers for this reason

Reply:
since Jenkins doesn't break, we don't necessarily need to bump it up but it would be nice to have working ITs for all OSes

Reply:
That's what I was wondering about. Sure, let's upgrade to the latest then